virttest/qemu_devices/qdevices.py

```diff
@@ -1586,7 +1586,8 @@ def get_supported_options(self, backend):
         elif backend in ["socket"]:
             common_opts += ["server", "wait", "reconnect"]
             special_opts = ["host", "port", "to", "ipv4",
-                            "ipv6", "nodelay", "path"]
+                            "ipv6", "nodelay", "path",
+                            "abstract", "tight"]
```

Comment:
Need be updated too,  for unix_socket only

Reply:
> Need be updated too, for unix_socket only

update and thanks.